README.md

```diff
@@ -83,7 +83,7 @@ To compile and run all of the examples in a row, you can use `make runExamples`
 - [hidefromkgb's 'mac_load' repository](https://github.com/hidefromkgb/mac_load#objective-c-gui-app-example) - the backend for some of the important parts of Silicon (such as defining Objective-C types and functions).
 - Apple - all of the headers from [include/Silicon/headers](include/Silicon/headers/) have been directly copied and modified to not have any Objective-C shenanigans in them for them to be compiled in C.
 - EimaMei - Original creator of Silicon
-- [Colleague Riley](https://github.com/ColleagueRiley) - Creator of the orginal single-header version of Silicon (silicon.h) and co-author of silicon
+- [Colleague Riley](https://github.com/ColleagueRiley) - Creator of the original single-header version of Silicon (silicon.h) and co-author of silicon
```

Comment:
Fix the capitalization, that being changing 'Creator' to be in lowercase and 'silicon' to have its first letter capitalized. Also a dot is required at the end of the sentence.